build.gradle.kts

```diff
@@ -31,29 +30,29 @@ application {
 dependencies {
   implementation("org.hamcrest:hamcrest:2.2")
   implementation("javax.help:javahelp:2.0.05")
-  implementation("com.fifesoft:rsyntaxtextarea:3.1.2")
+  implementation("com.fifesoft:rsyntaxtextarea:3.1.3")
   implementation("net.sf.nimrod:nimrod-laf:1.2")
   implementation("org.drjekyll:colorpicker:1.3")
   implementation("at.swimmesberger:swingx-core:1.6.8")
   implementation("org.scijava:swing-checkbox-tree:1.0.2")
-  implementation("org.slf4j:slf4j-api:1.7.30")
-  implementation("org.slf4j:slf4j-simple:1.7.30")
-  implementation("com.formdev:flatlaf:1.2")
-  implementation("commons-cli:commons-cli:1.4")
+  implementation("org.slf4j:slf4j-api:1.7.32")
+  implementation("org.slf4j:slf4j-simple:1.7.32")
+  implementation("com.formdev:flatlaf:1.6.4")
+  implementation("commons-cli:commons-cli:1.5.0")
   implementation("org.apache.commons:commons-text:1.9")
 
   // NOTE: Do not upgrade the jflex version. Later versions do not work.
   compileOnly("de.jflex:jflex:1.4.1")
-  compileOnly("org.jetbrains:annotations:22.0.0")
+  compileOnly("org.jetbrains:annotations:23.0.0")
```

Comment:
I believe this one can be safely removed as we do not seem to use it.

Reply:
Annotations removed.